tests/Makefile

```diff
@@ -203,8 +204,8 @@ killall:
 
 NUNIT_MSBUILD_DIR=$(TOP)/packages/NUnit.Runners.2.6.4/tools/lib
 test-ios-tasks:
-	$(SYSTEM_XBUILD) $(TOP)/msbuild/Xamarin.MacDev.Tasks.sln
-	cd $(NUNIT_MSBUILD_DIR) && $(SYSTEM_MONO) ../nunit-console.exe ../../../../msbuild/tests/bin/Xamarin.iOS.Tasks.Tests.dll -xml=TestResults_Xamarin.iOS.Tasks.Tests.xml -labels $(TEST_FIXTURE) || touch .failed-stamp
+	$(SYSTEM_XBUILD) /fl "/flp:v=diag;LogFile=msbuild-macdev.log" $(TOP)/msbuild/Xamarin.MacDev.Tasks.sln
```

Comment:
Looks like you've added debug spew here. Is that needed?

Reply:
err.. nope :)